### PR TITLE
Adding Graph panels

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Working for k8s-netperf and kube-burner workloads.

- k8s-netperf will show `TCP-STREAM` throughput.
- kube-burner will show `P99-PodReady Latency` (seconds)

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by chentex — APPROVED** [review]
LGTM

File: backend/app/api/endpoints/graph.py
Comment (paigerube14) [inline backend/app/api/endpoints/graph.py:196]: should the above lines of setting query and this if statement be in an if/else?

Comment (chentex) [reply backend/app/api/endpoints/graph.py:196]: I understood it as, this is the default query if we get nodeCount data, replace with this query

Comment (paigerube14) [reply backend/app/api/endpoints/graph.py:196]: Understood but why set it to begin with if we know we are going to overwrite. Just preference, don't need to hold up merging

Comment (jtaleric) [reply backend/app/api/endpoints/graph.py:196]: yeah... it only changes for kube-burner queries -- to include the number of workers and masters...

**Comment by jtaleric** [comment]
I have some changes in the hopper... let me wrap those up before we merge @chentex

**Comment by jtaleric** [comment]
@chentex alright, lemme know your final thoughts.

**Review by chentex — APPROVED** [review]

**Pull request merged**
